atlasdb-container-test-utils/build.gradle

```diff
@@ -41,3 +41,7 @@ dependencies {
         because 'allows JUnit 3 and JUnit 4 tests to run'
     }
 }
+
+test {
+    jvmArgs("--add-opens=java.base/java.lang=ALL-UNNAMED", "--add-opens=java.base/java.util=ALL-UNNAMED")
+}
```

Comment:
This solves `CassandraEnvironmentTest`

Reply:
Again, not so nice. We can use an external package to get around this issue. I briefly tried to implement a solution that does not depend on external packages and is compatible with JDK 17, but it did not work.

Reply:
As in the other one with `--add-opens`: it would be good to leave some context here